sled-storage/src/manager_test_harness.rs

```diff
@@ -83,10 +83,9 @@ pub struct StorageManagerTestHarness {
 impl Drop for StorageManagerTestHarness {
     fn drop(&mut self) {
         if let Some(vdev_dir) = self.vdev_dir.take() {
-            eprintln!(
+            eprint!(
```

Comment:
This change is split out as a separate PR here: https://github.com/oxidecomputer/omicron/pull/7111

It's just a nice-to-have for incremental testing.

Reply:
Merged now; resolving this comment